xbmc/music/Song.h

```diff
@@ -72,6 +72,11 @@ class CSong: public ISerializable
     return false;
   }
 
+  //Get artist names as a string vector from the vector of artist credits
```

Comment:
At least 3 lines of comment for one simple function does not seem very helpful to me in this case. Do you really want it? Commenting is very diverse through this section of code, I am not really breaking any standard that I can see. Do you want me to change it again?